lib/api.js

```diff
@@ -77,6 +78,7 @@ module.exports = settings => {
   flow.hook('create', hooks.updateAction(settings));
   flow.hook('create', hooks.submissionMeta(settings));
   flow.hook('create', hooks.create(settings));
+  flow.hook('create', hooks.modelData(settings));
```

Comment:
I don't think this should be here, since it's already applied on L77
```suggestion
```

Reply:
ah - no this needs to happen after create, as the training pil won't exist until then. I left the other one before create as I didn't know what it would break elsewhere

Reply:
Let's find out then and remove the other. Running the same hook twice is nasty.